README.md

```diff
@@ -73,3 +73,15 @@ All development work is done on local copies of the code base.  Follow the steps
 - Register for an [Earthdata Login account](https://sit.urs.earthdata.nasa.gov/) for the SIT environment.
 
 - Request ACL permissions to access the tool in the SIT environment by emailing [Earthdata Support](mailto:support@earthdata.nasa.gov). In order to Ingest collections into the tool, you may need Admin or Arc Curator permissions as well.
+
+## Other Known Issues
+
+During the RAILS 5.2 upgrade, there was an issue with the CSRF authenticity tokens.   Namely, this specific 
+workflow:  if a user clicks See Review Details, then clicks Curation Home, then clicks Revert Record,
+they will get a Invalid Authenticity Token.   Workaround is to tell form_with it should not auto include the
+token, rather we should explicitly include it ourselves. i.e.,
```

Comment:
is it worth mentioning here that we only need to include it for non get requests?